source/module_lr/potentials/pot_hxc_lrtd.h

```diff
@@ -15,8 +17,9 @@ namespace LR
         enum SpinType { S1 = 0, S2_singlet = 1, S2_triplet = 2, S2_updown = 3 };
```

Comment:
could you explain what is the meaning of S2_undown?

Reply:
It calculates $f_{\uparrow\uparrow}\rho_\uparrow$, $f_{\uparrow\downarrow}\rho_\downarrow$, $f_{\downarrow\uparrow}\rho_\uparrow$ or $f_{\downarrow\downarrow}\rho_\downarrow$ according to the spin of the input charge and the output potential.

Reply:
@maki49 I see, but I think the `SpinType` cannot be evaluated as a good name. You can ask @dyzheng for more appropriate nomenclature, or, have you ever investigated how other softwares name them?